lp-core-platform/lp-cp-xwiki-implementation/src/main/java/eu/learnpad/core/impl/mv/XwikiController.java

```diff
@@ -102,6 +104,17 @@ public synchronized void updateBridgeInterface (BridgeInterface bi){
 	@Override
 	public void notifyVerification(String verificationProcessId)
 			throws LpRestException {
+
+	    VerificationStatus status = this.bridge.getVerificationStatus(verificationProcessId);
+	    //TODO: show the status.getStatus() somewhere in the wiki for the verification with id verificationProcessId?
+	    
+	    if(!status.getStatus().equals(StatusType.COMPLETED)){
+	        //A severe error occurred in the verification phase that is terminated unexpectedly
+	        //The verification results in this case is null. If this.bridge.getVerificationResult will be called an exception will rise up
+	        //Notify somewhere the status of the verification id? or obtain the modelSet Id from the verification id and notify the verification status of the modelset id?
+	        return;
+	    }
+	    
```

Comment:
@damianofalcioni, what is supposed to happen here? Actually these part of the if statement looks not completed. Can you confirm?